src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -4827,7 +4843,7 @@ class CsvFileFormatTypeParameters(BaseSegment):
         Sequence(
             "NULL_IF",
             Ref("EqualsSegment"),
-            Bracketed(Delimited(Ref("QuotedLiteralSegment"), optional=True)),
+            OptionallyBracketed(Delimited(Ref("QuotedLiteralSegment"), optional=True)),
```

Comment:
Is this change supposed to be here? And, if so, can we add a test for it?

Reply:
Checking the docs, I don't think this is allowed https://docs.snowflake.com/en/sql-reference/sql/alter-file-format

I'll revert it.